### PR TITLE
Add syslog_tls ingestor

### PR DESCRIPTION
**Pull request by mrdavidlaing**
So messages shipped via syslog can be encrypted with SSL

### CODE REVIEW

**Comment by mrdavidlaing** [comment]
@dpb587 - Thoughts on how I've implemented this - as an additional `input` in the ingestor_syslog job

File: jobs/ingestor_syslog/templates/config/syslog_to_redis.conf.erb [hunk 1]
```diff
@@ -9,6 +10,18 @@ input {
 		host => "0.0.0.0"
 		port => "<%= p("logstash_ingestor.syslog.port") %>"
 	}
+	<% end %>
+
+	<% if p("logstash_ingestor.syslog_tls.port") %>
+    tcp {
```
Comment (dpb587) [inline jobs/ingestor_syslog/templates/config/syslog_to_redis.conf.erb:16]: Indent slightly off.

**Comment by dpb587** [comment]
This looks good to me.

**Comment by mrdavidlaing** [comment]
Cool - I'll work through checking that it works with nxlog tomorrow

**Pull request merged**